tutorials/fundamentals/first-child.md

```diff
@@ -0,0 +1,38 @@
+The `firstChild` property returns the first child node of a node.
+If you wish to change the text of that property, you need to use the `firstElementChild` property instead.
+The `firstChild` property returns a node, which could be of type `text` or `element`.
+When HTML is written like below, it will return type `text`, as the whitespace between the `ul` tag and `li` tag cause that issue.
+Any whitespace creates this issue and so the HTML would need to be written on one continuous line in order for it to return element.
+
+<ul id="example">
+<li>Gas</li>
+<li>Food</li>
+</ul>
+<script>
+const list = document.querySelector('#example');
+list.firstElementChild.innerHTML = 'Water';
+</script>
+
+```html
+<ul id="example">
```

Comment:
I think I see why you're using `firstElementChild` here. The newline here causes HTML to insert a text node. If you do `<ul id="example"><li>Gas</li><li>Food</li></ul>` all on one line, `firstChild` will correctly return the `li`.

Add a section about how HTML whitespace affects `firstChild`, and make a note about `firstElementChild` here. This issue with newlines affecting `firstChild` happened to me this week too :)